Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    background: var(--white);
+    height: 100px;
+}
+
+header div {
```

Comment:
Do we need this div? What is it doing that the h1 cannot do?

Reply:
Yes, definitely. I removed it, thanks